src/app/options.h

```diff
@@ -1,5 +1,7 @@
 /*
- * Bittorrent Client using Qt4 and libtorrent.
+ * Bittorrent Client using Qt and libtorrent.
+ * Copyright (C) 2016  Eugene Shalygin
```

Comment:
At the time of writing I did not want to leave e-mail there.  Probably I'll insert the e-mail during the next round of changes.